### PR TITLE
Test for CASSANDRA-9765

### PR DESCRIPTION
**Pull request by stef1927**
A new test for CASSANDRA-9765.

### CODE REVIEW

File: bootstrap_test.py [hunk 1]
```diff
@@ -287,3 +288,49 @@ def local_quorum_bootstrap_test(self):
...
+
+        session = self.patient_cql_connection(node1)
+        original_rows = list(session.execute("SELECT * FROM %s" % (stress_table,)))
+
+        # Add a new node, bootstrap=True ensures that it is not a seed
+        node2 = new_node(cluster, bootstrap=True)
+        node2.start(wait_for_binary_proto=True)
+
+        session = self.patient_cql_connection(node2)
+        self.assertEquals(original_rows, list(session.execute("SELECT * FROM %s" % (stress_table,))))
```
Comment (ptnapoleon) [inline bootstrap_test.py]: Can you make this string format, and the other at line 318, match the python 3 syntax, by using "".format()? Thanks

**Comment by mambocab** [comment]
Looks great, passes on stef1927/9765-2.0. Thanks a lot, @stef1927!

File: bootstrap_test.py (hunk 1 above)
Comment (stef1927) [inline bootstrap_test.py]: Copy and paste from another test, sorry about it.

**Comment by stef1927** [comment]
Fixed string format to match python 3 syntax, see second commit.

**Comment by mambocab** [comment]
LGTM, thanks @stef1927!

**Comment by stef1927** [comment]
I added more tests but they are still not final, so hold the merge.

**Comment by mambocab** [comment]
Yep, I'll wait for you to merge.

**Pull request merged**